React/wbooks/src/app/components/RouteHandler/index.js

```diff
@@ -10,7 +10,7 @@ class RouteHandler extends Component {
   };
 
   isPublicRequest = props => {
-    return props.location.pathname === '' || props.location.pathname === '/';
+    return props.location.pathname === '' || props.location.pathname === '/' || props.location.pathname === '/signup';
```

Comment:
Are u sure that `props.location.pathname === ''` is a valid  value? I think that `pathname` never can be `''`